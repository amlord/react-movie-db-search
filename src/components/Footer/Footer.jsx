import React from 'react';
import styled from 'styled-components';
import Wrapper from 'components/Wrapper';

const StyledFooter = styled.footer`
  background-color: #081c24;
  color: #fff;
`;

const Footer = () => (
  <StyledFooter>
    <Wrapper><small>By Andrew Lord &copy; 2018</small></Wrapper>
  </StyledFooter>
);

export default Footer;
