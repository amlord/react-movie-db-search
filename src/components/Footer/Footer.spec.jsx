import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer', () => {
  it('displays correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
