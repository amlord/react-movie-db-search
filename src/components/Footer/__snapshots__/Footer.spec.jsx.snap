// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer displays correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Wrapper>
        <small>
          By Andrew Lord © 2018
        </small>
      </Wrapper>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <small>
          By Andrew Lord © 2018
        </small>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "By Andrew Lord © 2018",
        },
        "ref": null,
        "rendered": "By Andrew Lord © 2018",
        "type": "small",
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Wrapper>
          <small>
            By Andrew Lord © 2018
          </small>
        </Wrapper>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <small>
            By Andrew Lord © 2018
          </small>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "By Andrew Lord © 2018",
          },
          "ref": null,
          "rendered": "By Andrew Lord © 2018",
          "type": "small",
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
